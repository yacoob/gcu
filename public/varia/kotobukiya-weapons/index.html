<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <!--[if lt IE 9]>
  <script src='https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js'></script>
  <script src='https://oss.maxcdn.com/respond/1.4.2/respond.min.js'></script>
  <![endif]-->
  <link href='/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='//syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130331_153330.jpg' rel='image_src'/>
  <meta property='og:image' content='https://syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130331_153330.jpg'/>
  <meta name='description' content='tactical grace presents: Kotobukiya weapons'/>
  <link href='http://gcu.tactical-grace.net/varia/kotobukiya-weapons/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>Kotobukiya weapons | tactical grace</title>
  <meta content='Kotobukiya weapons | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/varia/kotobukiya-weapons/'>Kotobukiya weapons</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a href='/varia/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a href='/varia/kotobukiya-tachikoma/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
    <a href='/varia/odaiba-rx-78-2/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-right'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2013-04-24'>2013-04-24</h2>
  </div>
  <div class='row voffset-small'>
  <div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130325_230822.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-04-24/s400-p/IMG_20130325_230822.jpg' title='Need to swat down a tank? Or a satellite? We have a tool for you!' alt='Need to swat down a tank? Or a satellite? We have a tool for you!' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130331_153330.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-04-24/s400-p/IMG_20130331_153330.jpg' title='One of those shields that you need to dig in in order for it to be useful.' alt='One of those shields that you need to dig in in order for it to be useful.' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130331_153700.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-04-24/s400-p/IMG_20130331_153700.jpg' title='Yeah, that shield... transforms. Extends. Moves around and does... something.' alt='Yeah, that shield... transforms. Extends. Moves around and does... something.' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130331_195020.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-04-24/s400-p/IMG_20130331_195020.jpg' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130331_195559.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-04-24/s400-p/IMG_20130331_195559.jpg' title='Might go well with that MS-07B3 Gouf I&#39;m thinking about.' alt='Might go well with that MS-07B3 Gouf I&#39;m thinking about.' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130407_203018.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-04-24/s400-p/IMG_20130407_203018.jpg' title='It&#39;s spring-loaded. You can punch things.' alt='It&#39;s spring-loaded. You can punch things.' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130407_203107.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-04-24/s400-p/IMG_20130407_203107.jpg' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130407_212118.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-04-24/s400-p/IMG_20130407_212118.jpg' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130407_214652.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-04-24/s400-p/IMG_20130407_214652.jpg' title='Yup, the chain actually moves.' alt='Yup, the chain actually moves.' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130408_184222.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-04-24/s400-p/IMG_20130408_184222.jpg' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130408_184259.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-04-24/s400-p/IMG_20130408_184259.jpg' title='Keep calm and pierce heavens with your drill!' alt='Keep calm and pierce heavens with your drill!' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2013-04-24/s1920/IMG_20130408_221506.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2013-04-24/s400-p/IMG_20130408_221506.jpg' title='The underlying mechanism is the same as for the fist.' alt='The underlying mechanism is the same as for the fist.' />
    </a>
  </div>
</div>
  </div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='http://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link href='/css/jquery.fancybox.css' rel='stylesheet'>
<script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js'></script>
<script src='/js/jquery.fancybox.min.js'></script>
<script src='/js/jquery.fancybox-media.js'></script>
<script src='/js/screenfull.min.js'></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
$(document).ready(gcu.postPageHandler);
</script>
</body></html>