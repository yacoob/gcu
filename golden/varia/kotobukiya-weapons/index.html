<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <link href='https://cdn.jsdelivr.net/npm/bootstrap3@3/dist/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='/photos/full/gcu-00162.jpg' rel='image_src'/>
  <meta property='og:image' content='https://gcu.tactical-grace.net/photos/full/gcu-00162.jpg'/>
  <meta name='twitter:card' content='summary_large_image'/>
  <meta name='twitter:image' content='https://gcu.tactical-grace.net/photos/full/gcu-00162.jpg'/>
  <meta name='description' content='tactical grace presents: Kotobukiya weapons'/>
  <meta name='twitter:title' content='tactical grace presents: Kotobukiya weapons'/>
  <link href='https://gcu.tactical-grace.net/varia/kotobukiya-weapons/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>Varia: Kotobukiya weapons | tactical grace</title>
  <meta content='Varia: Kotobukiya weapons | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/varia/kotobukiya-weapons/'>Varia: Kotobukiya weapons</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a data-toggle='tooltip' title='main page' href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a data-toggle='tooltip' title='contact the author' href='https://goo.gl/forms/iJEsfLjBy1FE86aQ2'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-envelope'></span></button></a>
    <a data-toggle='tooltip' title='grade page' href='/varia/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a data-toggle='tooltip' title='previous kit' href='/varia/kotobukiya-tachikoma/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
    <a data-toggle='tooltip' title='next kit' href='/varia/odaiba-rx-78-2/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-right'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2013-04-24'>2013-04-24</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00154.jpg' data-sub-html='Need to swat down a tank? Or a satellite? We have a tool for you!'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00154.jpg' alt='Need to swat down a tank? Or a satellite? We have a tool for you!' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00162.jpg' data-sub-html='One of those shields that you need to dig in in order for it to be useful.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00162.jpg' alt='One of those shields that you need to dig in in order for it to be useful.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00163.jpg' data-sub-html='Yeah, that shield... transforms. Extends. Moves around and does... something.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00163.jpg' alt='Yeah, that shield... transforms. Extends. Moves around and does... something.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00164.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00164.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00165.jpg' data-sub-html='Might go well with that MS-07B3 Gouf I&#39;m thinking about.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00165.jpg' alt='Might go well with that MS-07B3 Gouf I&#39;m thinking about.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00166.jpg' data-sub-html='It&#39;s spring-loaded. You can punch things.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00166.jpg' alt='It&#39;s spring-loaded. You can punch things.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00167.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00167.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00168.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00168.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00169.jpg' data-sub-html='Yup, the chain actually moves.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00169.jpg' alt='Yup, the chain actually moves.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00170.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00170.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00171.jpg' data-sub-html='Keep calm and pierce heavens with your drill!'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00171.jpg' alt='Keep calm and pierce heavens with your drill!' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00172.jpg' data-sub-html='The underlying mechanism is the same as for the fist.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00172.jpg' alt='The underlying mechanism is the same as for the fist.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
</div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='https://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/css/lightgallery.min.css" />
<script src="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/js/lightgallery.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-thumbnail.js@1/dist/lg-thumbnail.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-zoom.js@1/dist/lg-zoom.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-video.js@1/dist/lg-video.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-fullscreen.js@1/dist/lg-fullscreen.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-autoplay.js@1/dist/lg-autoplay.min.js"></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
document.addEventListener('DOMContentLoaded', gcu.postPageHandler);
</script>
</body></html>